docs/concepts/fields.md

```diff
@@ -41,6 +41,7 @@ class User(BaseModel):
 ## Using `Annotated`
 
 The `Field` function can also be used together with `Annotated`.
+For versions of Python prior to 3.9, `typing_extensions.Annotated` can be used.
```

Comment:
Why do you think we should mention `typing_extensions`?
```suggestion
```

I don't mind having it, but we may want to think about the other places in the documentation where we have analogous cases?

Reply:
Yeah, I'm fine with removing. If we see any issues crop up with people asking about import errors with `Annotated`, then we can do a more comprehensive change to the docs